### ISSUE TITLE
Upgrade to AngularJS 1.5

### ISSUE THREAD

**Issue by jdubois** [jhipster/generator-jhipster#2808]
See http://angularjs.blogspot.fr/2016/02/angular-150-ennoblement-facilitation.html?m=1

- this should be as part of our 3.0 redesign
- here's some good work for my trainees cc @PierreBesson

**Comment by deepu105** [comment]
I dont think this will break anything for us.May be we can rewrite our directives with the new component helper

**Comment by PierreBesson** [comment]
Wow, this looks a lot like React ! We could start working on this next week when William (the other JHipster intern) arrives. He told me he had some AngularJS experience.

**Comment by jdubois** [comment]
Yes that's the idea :-)
We'll do a team briefing on Monday next week.

**Comment by wmarques** [comment]
Hi, I have started to work on this issue. Check this [commit](https://github.com/jhipster/generator-jhipster/commit/a39de14782e1bebc96102abb57d82cec6f321516).
I have tested the component "alert-error" I modified (click on Delete User in administration) and the console gives me no bug. But I have several bugs when running "gulp test", maybe a dependency in Karma or in some bower library is causing those bugs ?
